src/sentry/seer/anomaly_detection/delete_rule.py

```diff
@@ -26,6 +21,8 @@ def delete_rule_in_seer(alert_rule: "AlertRule") -> bool:
     """
     Send a request to delete an alert rule from Seer. Returns True if the request was successful.
     """
+    from sentry.seer.anomaly_detection.utils import SEER_ANOMALY_DETECTION_CONNECTION_POOL
```

Comment:
should this be a regular import instead of type checking only? (It looks like it's being used as a default value on line 36).